app/default_overrides.go

```diff
@@ -259,19 +260,13 @@ func DefaultEvidenceParams() tmproto.EvidenceParams {
 func DefaultConsensusConfig() *tmcfg.Config {
 	cfg := tmcfg.DefaultConfig()
 	// Set broadcast timeout to be 50 seconds in order to avoid timeouts for long block times
-	// TODO: make TimeoutBroadcastTx configurable per https://github.com/celestiaorg/celestia-app/issues/1034
 	cfg.RPC.TimeoutBroadcastTxCommit = 50 * time.Second
 	cfg.RPC.MaxBodyBytes = int64(8388608) // 8 MiB
 
-	cfg.Mempool.TTLNumBlocks = 5
-	cfg.Mempool.TTLDuration = time.Duration(cfg.Mempool.TTLNumBlocks) * appconsts.GoalBlockTime
-	// Given that there is a stateful transaction size check in CheckTx,
-	// We set a loose upper bound on what we expect the transaction to
-	// be based on the upper bound size of the entire block for the given
-	// version. This acts as a first line of DoS protection
-	upperBoundBytes := appconsts.DefaultSquareSizeUpperBound * appconsts.DefaultSquareSizeUpperBound * share.ContinuationSparseShareContentSize
-	cfg.Mempool.MaxTxBytes = upperBoundBytes
-	cfg.Mempool.MaxTxsBytes = int64(upperBoundBytes) * cfg.Mempool.TTLNumBlocks
+	cfg.Mempool.TTLNumBlocks = 12
+	cfg.Mempool.TTLDuration = 75 * time.Second
+	cfg.Mempool.MaxTxBytes = 7_897_088
+	cfg.Mempool.MaxTxsBytes = 39_485_440
```

Comment:
These 3 lines just set the values in a more clear way. The actual values are identical before + after this PR.

Reply:
Can we have an issue to bump this value past 8MB

Reply:
isn't this limited to 2MB anyway?